lib/tapioca/dsl/compilers/frozen_record.rb

```diff
@@ -65,7 +65,7 @@ module Compilers
       class FrozenRecord < Compiler
         extend T::Sig
 
-        ConstantType = type_member { { fixed: T.class_of(::FrozenRecord::Base) } }
+        ConstantType = type_member { { fixed: T.all(T.class_of(::FrozenRecord::Base), Extensions::FrozenRecord) } }
```

Comment:
Is there an alternative way to explicitly tell Sorbet that `T.class_of(::FrozenRecord::Base).is_a?(Extensions::FrozenRecord)`?

Reply:
I don't think so.